docs/sources/operators-guide/getting-started/_index.md

```diff
@@ -103,7 +103,7 @@ In a terminal, run one of the following commands:
 - Using Docker:
 
   ```bash
-  docker run --rm --name mimir --publish 9009:9009 --volume "$(pwd)"/demo.yaml:/etc/mimir/demo.yaml grafana/mimir:${MIMIR_LATEST} --config.file=/etc/mimir/demo.yaml
+  docker run --rm --name mimir --publish 9009:9009 --volume "$(pwd)"/demo.yaml:/etc/mimir/demo.yaml grafana/mimir:latest --config.file=/etc/mimir/demo.yaml
```

Comment:
`MIMIR_LATEST` environment variable doesn't exist. Since in the previous step we ask to run `docker pull grafana/mimir:latest`, here we can just use `latest`.